articles/architecture-scenarios/b2b/b2b-architecture.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Architecture
+description: Understand how Auth0 lets you create Auth0 tenants for one or more environments
+toc: true
+topics:
+    - b2b
+    - iam
+    - tenants
+contentType: concept
+useCase:
+  - tenant-architecture
+---
+
+# Architecture
+
+<%= include('../_includes/_architecture-intro.md') %>
+
+<%= include('../_includes/_bp-sso-training.md') %>
+
+## Design considerations
+
+<%= include('../_includes/_architecture-design-considerations.md') %>
+* How can I align the structure and domain of my customers' organization with my Auth0 deployment?
```

Comment:
I think this might need another space after the *, it is not showing up right in the end result.

Reply:
Actually an extra space doesn't look like it is going to help...  Maybe need a newline before the list...

Reply:
New line seems to works.